README.rst

```diff
@@ -112,3 +112,5 @@ Attribution
 Please cite `Murray, Power and Robotham (2013)
 <https://arxiv.org/abs/1306.6721>`_ and/or https://ascl.net/1412.006 (whichever is more appropriate) if you find this
 code useful in your research. Please also consider starring the GitHub repository.
+At last it is one of the finest calculators whiuch helps you to perform calculation at fast speed .
```

Comment:
Thank you for the positive feedback! I don't think I can put this in my readme though :-)